app/src/main/java/comp_sci_squad/com/github/url_irl/MainActivity.java

```diff
@@ -242,9 +157,28 @@ public void onClick(View v) {
         }
     };
 
+    /**
```

Comment:
Why does this move

Reply:
It seemed to be Android Studio rearranging funtions and classes based on Alphabetical order. For now, I'll restore it and we can create a new PR to re-arrange codes more cleanly